internal/nostr/nostr.go

```diff
@@ -277,7 +277,7 @@ func (svc *Service) PublishHandler(c echo.Context) error {
 	}
 
 	svc.Logger.WithFields(logrus.Fields{
-		"request_event_id":  requestData.SignedEvent.ID,
+		"event_id":  requestData.SignedEvent.ID,
```

Comment:
below there is a `request_event_id` and now here it's `event_id` 
check that all those are consistent.

Reply:
Yeah I removed this because an `event_id` is a `request_event_id` only in the case of nip47, here it is a normal publish so it can be any event

Reply:
Rest all should be consistent now ✅